config-overrides.js

```diff
@@ -92,12 +92,11 @@ function rewireBabel(config, _env) {
   const loader = getBabelLoader(config.module.rules);
 
   if (!loader) {
-    console.warn('babel-loader not found'); // eslint-disable-line no-console
```

Comment:
I'd like to keep this in here.  I copied this from react-app-rewired actually, they do the same thing whenever `getBabelLoader` can't find the babel config.  It will help with debugging any users having issues in the future.